### PR TITLE
[update] Added detailed instructions including screenshots to email/providers, small edits to index

### PR DESCRIPTION
**Pull request by Catografix**
Added a "testing only" disclaimer to the index doc. Added links to supported-provider signup pages to providers doc.

### CODE REVIEW

File: articles/email/index.md [hunk 1]
```diff
@@ -21,25 +22,25 @@ Users will see the sender's address in the **From Address** field when receiving
...
+You must add the [Sender Policy Framework (SPF)](http://en.wikipedia.org/wiki/Sender_Policy_Framework) and [DomainKeys Identified Mail (DKIM)](http://en.wikipedia.org/wiki/DKIM) DNS records to your domain's zone file to allow Auth0 to send digitally-signed emails on your behalf. Without these records, the emails may end up in your users' junkmail folders. Additionally, your users may see the following as the **From Address**:
 
 `MyApp support@mail128-21.atl41.mandrillapp.com on behalf of MyApp support@fabrikamcorp.com`
 
 #### SPF Configuration
 
-You can configure the SPF by adding a TXT record to your domain. You should set the host name to `@`, or leave it empty, depending on the provider. If this is the first time you are setting up an SPF record, you will need to declare it as follows:
+You can configure the SPF by adding a TXT record to your domain's zone file. You should set the host name to `@`, or leave it empty, depending on the provider. If this is the first time you are setting up an SPF record, you will need to declare it as follows:
```
Comment (Catografix) [inline articles/email/index.md:45]: Im pretty sure it's the zone file that this TXT record is added to, please verify.

Comment (ntotten) [inline articles/email/index.md:45]: Yeah, i believe this is right word. :+1:

File: articles/email/providers.md [hunk 2]
```diff
@@ -1,60 +1,119 @@
-# Using your own SMTP Provider for sending emails
+# Use your own SMTP Email Provider
 
-Instead of using Auth0's infrastructure for email communication with your users the Auth0 API v2 also allows you to configure your own provider. This allows you to better monitor and troubleshoot the email communication.
+The Auth0 API v2 allows you to configure your own SMTP email provider instead of using Auth0's infrastructure for email communication with your users. This enables you to more completely manage, monitor and troubleshoot your email communications.
```
Comment (ntotten) [inline articles/email/providers.md]: I would remove the "Auth0 API v2" bit. You can configure it via the dashboard or the API - either way the "how" you actually do the configuration isnt really relevant in the intro.

File: articles/email/providers.md [hunk 3]
```diff
@@ -1,60 +1,119 @@
-# Using your own SMTP Provider for sending emails
+# Use your own SMTP Email Provider
```
Comment (ntotten) [inline articles/email/providers.md:1]: One thing I forgot to mention is that we are no longer recommending (or supporting) the use of Auth0's email provider for production use - You now must use your own custom provide in production. So Lets add something that nots that the auth0 provide is for testing, etc. and you should use your preferred provider before you switch to production.

I would also recommend we add links showing where and how users can sign up to each of our supported services. We dont need to write those docs, the other providers should have them.

**Pull request merged**